blocks/comparison/comparison.js

```diff
@@ -0,0 +1,41 @@
+import { getLibs } from '../../scripts/utils.js';
+
+/**
+ * @param {Element} el
+ */
+const init = async (el) => {
+  const { createTag } = await import(`${getLibs()}/utils/utils.js`);
+
+  const rows = Array.from(el?.querySelectorAll(':scope > div'));
+  const headers = Array.from(rows.shift().children);
+
+  const headersRow = createTag('tr');
+  headers.forEach((cell) => {
+    headersRow.appendChild(createTag('th', { scope: 'col' }, cell));
+  });
+  const thead = createTag('thead', {}, headersRow);
+
+  const tbody = createTag('tbody');
```

Comment:
Variables should be grouped together without spaces. Airbnb styleguide says to have a newline after variables but I don't always see that observed in Milo, so I think having them all together is fine too if you prefer
```suggestion
  const { createTag } = await import(`${getLibs()}/utils/utils.js`);
  const rows = Array.from(el?.querySelectorAll(':scope > div'));
  const headers = Array.from(rows.shift().children);
  const headersRow = createTag('tr');

  headers.forEach((cell) => {
    headersRow.appendChild(createTag('th', { scope: 'col' }, cell));
  });

  const thead = createTag('thead', {}, headersRow);
  const tbody = createTag('tbody');
```

Reply:
Updated to fit Airbnb styleguide.